docs/plugins/virtual_environment_docs/article-6-survey.md

```diff
@@ -0,0 +1,118 @@
+# Tools and tips when building napari plugins: Survey 
```

Comment:
I'm not sure how useful this Q&A will be outside of the context of the workshop. I would vote to omit this personally. I believe everything here is covered in other docs. Curious what others think.

Reply:
It seemed like it would be useful to beginners.

Reply:
Yeah there's a lot of useful info here. First thing I guess is I would maybe call it a Q&A rather than a survey? And I would make the questions either their own headings or dropdown admonitions so that the page is easier to browse at a glance. Right now I feel like the questions get lost in the doc.

Reply:
Unless you have a strong objection, I'm going to leave this as is.

Reply:
@Nadalyn-CZI I think the formatting here is important, so it will need to be fixed before merge. Let me know if the suggestions are unclear.